pages/common/newsboat.md

```diff
@@ -0,0 +1,16 @@
+# newsboat
+
+> An RSS/Atom feed reader for the text console.
+> More information: https://newsboat.org/.
+
+- First import feed urls from an OPML file
```

Comment:
Hello @dancek, I don't understand. Are these installation instructions? We focus the tldr pages into examples for commands usage.

Reply:
When you setup a feed reader, @navarroaxel, you generally import your list of feeds. These are stored in the standard OPML format :-)

To that end, this example is a great one to start with.

Reply:
Newsboat refuses to start at all until you have a list of feeds in `~/.newsboat/urls`. This is something like an installation instruction, yes, but it's also the main reason people would read the man page. I think it's useful.